plugins/inputs/mysql/v2/convert.go

```diff
@@ -77,6 +77,59 @@ func parseWsrepLatency(value sql.RawBytes) (interface{}, error) {
 	return result, nil
 }
 
+// ParseWsrepProviderOptions parses the given sql.RawBytes value into a map
+// containing all wsrep Provider options settings.
+func parseWsrepProviderOptions(value sql.RawBytes) (interface{}, error) {
+	parts := strings.Split(string(value), "; ")
+	result := make(map[string]interface{})
+	for _, setting := range parts {
+		key, data, found := strings.Cut(setting, " = ")
```

Comment:
Are we sure about the spaces? Maybe it's more robust to split by `;` and `=` and use `strings.TrimSpace` on the key and value before further processing? Otherwise this might break if someone on the MySQL side decides to eliminate the spaces...